githubtraining/training-manual
# Add an option for generating PDFs

When we started using docsify, we lost the ability to generate PDFs automatically. This PR allows us to generate one manually with minimal effort, and adds a PDF to the existing manual.

To test:
1. Check out to this branch
2. Run `docsify serve docs`
3. Open http://localhost:3000, ensure the link to the PDF works
4. Generate your own PDF by running `script/generate-pdf`
5. Open `docs/book.pdf` to ensure the PDF was generated.

@githubtraining/trainers I can 🚢 this with one 👍.

![manual-pdf](https://user-images.githubusercontent.com/16547949/43136386-8c9c7608-8f16-11e8-87e9-e2f50751144c.gif)

Closes https://github.com/github/services-training/issues/795.
